src/integration-tests/src/storage.rs

```diff
@@ -16,10 +16,92 @@ use crate::{Error, Result};
 use gax::exponential_backoff::{ExponentialBackoff, ExponentialBackoffBuilder};
 use gax::options::RequestOptionsBuilder;
 use gax::paginator::{ItemPaginator, Paginator};
+use gax::retry_policy::RetryPolicyExt;
 use std::time::Duration;
-use storage::model::Bucket;
+use storage_control::model::Bucket;
 
-pub async fn buckets(builder: storage::client::ClientBuilder) -> Result<()> {
+pub async fn objects(builder: storage::client::ClientBuilder) -> Result<()> {
+    // Enable a basic subscriber. Useful to troubleshoot problems and visually
+    // verify tracing is doing something.
+    #[cfg(feature = "log-integration-tests")]
+    let _guard = {
+        use tracing_subscriber::fmt::format::FmtSpan;
+        let subscriber = tracing_subscriber::fmt()
+            .with_level(true)
+            .with_thread_ids(true)
+            .with_span_events(FmtSpan::NEW | FmtSpan::CLOSE)
+            .finish();
+
+        tracing::subscriber::set_default(subscriber)
+    };
+
+    // Create a temporary bucket for the test.
+    let (control, bucket) = create_test_bucket().await?;
+
+    let client = builder.build().await?;
+
+    tracing::info!("testing insert_object()");
```

Comment:
Q: should we prefer this to `println!()` ?

Reply:
I think so, they show up in order with other tracing logs.